### PR TITLE
solved the problem

### PR DESCRIPTION
**Pull request by isohamnemesis**

### CODE REVIEW

File: First_Proj/Player.cpp [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ * Player.cpp
+ *
+ *  Created on: 14-Jan-2021
+ *      Author: sam
+ */
+
+#include "Player.h"
+
+Player::Player(string n, string i) {
+	// TODO Auto-generated constructor stub
```
Comment (gcnit) [inline First_Proj/Player.cpp:11]: Do not use single letter variable names

File: First_Proj/Ladder.h [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+#ifndef LADDER_H_
+#define LADDER_H_
+#include<iostream>
+#include<string>
+#include<bits/stdc++.h>
+using namespace std;
+
+class Ladder {
+	int startpt, endpt;
+public:
```
Comment (gcnit) [inline First_Proj/Ladder.h:17]: Could just be start, end

File: First_Proj/Service.cpp [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+	snlb.setsize(default_size);
+	srand(time(0));
+}
+
+int Service::DiceRoll(){
+
+	return (1 + (rand()%6) );
+}
+void Service::setPlayer(queue<Player> p){
+
```
Comment (gcnit) [inline First_Proj/Service.cpp:21]: setPlayers

File: First_Proj/Service.cpp [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+	unordered_map<string, int> playerp;
+	while(!p.empty())
+	{
+		Player k = p.front();
+		player.push(k);
+		playerp.emplace(make_pair(k.getname(), 0));
+		p.pop();
+	}
+
+	snlb.setplayerpeices(playerp);
```
Comment (gcnit) [inline First_Proj/Service.cpp:31]: Use Camel case

Comment (gcnit) [reply First_Proj/Service.cpp:31]: Also, pieces spelling is wrong.

playerp?

**Comment by isohamnemesis** [comment]
Okay. Is this good enough to get someone through?

**Pull request closed without merging**